cilium/cmd/bgp_peer_get.go

```diff
@@ -0,0 +1,70 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"sort"
+	"strconv"
+	"text/tabwriter"
+
+	"github.com/spf13/cobra"
+
+	"github.com/cilium/cilium/api/v1/models"
+	"github.com/cilium/cilium/pkg/command"
+)
+
+var BgpPeersCmd = &cobra.Command{
+	Use:   "peers",
+	Short: "List current state of all peers",
+	Long:  "List state of all peers defined in CiliumBGPPeeringPolicy",
+	Run: func(cmd *cobra.Command, args []string) {
+		res, err := client.Bgp.GetBgpPeers(nil)
+		if err != nil {
+			Fatalf("cannot get peers list: %s\n", err)
+		}
+
+		if command.OutputOption() {
+			if err := command.PrintOutput(res.GetPayload()); err != nil {
+				Fatalf("error getting output in JSON: %s\n", err)
+			}
+		} else {
+			printSummary(res.GetPayload())
+		}
+	},
+}
+
+func printSummary(peers []*models.BgpPeer) {
+	w := tabwriter.NewWriter(os.Stdout, 5, 0, 3, ' ', 0)
```

Comment:
Can you break out these magic numbers into consts (can be just above). I have no clue what these numbers do without looking up the docs.

Also, how do other CLI commands handle table generation? Is there a more generic solution or do we do things like this everywhere?

Reply:
I'll add some constants for it. I took reference from https://github.com/cilium/cilium/blob/master/cilium/cmd/bpf_auth_list.go#L78 

Looking at other commands, it seems like similar approach is used everywhere.